nist/int.go

```diff
@@ -37,8 +45,9 @@ var two = big.NewInt(2)
 // whose target is assumed never to change.
 //
 type Int struct {
-	V big.Int  // Integer value from 0 through M-1
-	M *big.Int // Modulus for finite field arithmetic
+	V  big.Int   // Integer value from 0 through M-1
+	M  *big.Int  // Modulus for finite field arithmetic
+	Bo ByteOrder // Endianness considered for this int
```

Comment:
Shouldn't it be BO rather than Bo, since it's an abbreviation and not a single word "bo"?